po/telescope.asteroidos.pot

```diff
@@ -0,0 +1,22 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Is this descriptive? :)

Reply:
can we move this po/ directory into the telegram subdirectory ?

Reply:
can we use Qt translations instead? I suspect we may have troubles with Weblate by having two different translation systems.

Reply:
Everything in the po folder is generated also the descriptive title ^^

Ye we could move the directory into the telescope subdirectory.

I'll check how good the default Qt translation works with ubuntu touch.